airflow/providers/amazon/aws/operators/batch.py

```diff
@@ -26,17 +26,18 @@
     - http://boto3.readthedocs.io/en/latest/reference/services/batch.html
     - https://docs.aws.amazon.com/batch/latest/APIReference/Welcome.html
 """
-from typing import TYPE_CHECKING, Any, Optional, Sequence
+import warnings
+from typing import Any, Optional, Sequence, TYPE_CHECKING
```

Comment:
```suggestion
from typing import TYPE_CHECKING, Any, Optional, Sequence
```

to fix static checks